samples/bluetooth/nrf_dm/app.overlay

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
If I understand correctly, for nRF52 devices the overlay must be added manually:
```
west build -p -b nrf52840dk_nrf52840 -- -DDTC_OVERLAY_FILE=app.overlay
```
and for nRF53 devices it's going to be applied automatically using the `boards` directory:
```
west build -p -b nrf5340dk_nrf5340_cpuapp   # no additional configuration needed
```
Is this difference documented anywhere?

Reply:
If `boards/<BOARD>.overlay` does not exist then `app.overlay` will be used.
So the `app.overlay` is automatically used for nRF52 devices.
We do not need to add the overlay manually. 

For the nRF53, an overlay from the `boards` directory (for the app core) or from the `child_image` directory (for the net core) is used.